src/shared/components/error-screen/url-errors/MissingGenomeError.tsx

```diff
@@ -0,0 +1,128 @@
+/**
+ * See the NOTICE file distributed with this work for additional information
+ * regarding copyright ownership.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+import { useNavigate } from 'react-router-dom';
+
+import * as urlFor from 'src/shared/helpers/urlHelper';
+
+import { useAppDispatch, useAppSelector } from 'src/store';
+
+import { deleteSpeciesAndSave } from 'src/content/app/species-selector/state/speciesSelectorSlice';
+
+import { getCommittedSpecies } from 'src/content/app/species-selector/state/speciesSelectorSelectors';
+
+import AlertButton from 'src/shared/components/alert-button/AlertButton';
+import { PrimaryButton } from 'src/shared/components/button/Button';
+import ButtonLink from 'src/shared/components/button-link/ButtonLink';
+
+import type { CommittedItem } from 'src/content/app/species-selector/types/species-search';
+
+import styles from './UrlError.scss';
+
+/**
+ * Consider possible scenarios why a genome may be "missing":
+ * - the genome existed, but has now been retired
+ * - the genome never existed; the user entered a url with an invalid genome id
+ */
+
+type Props = {
+  genomeId: string;
```

Comment:
Would it be better to use a generic term like `identifier` instead?

Reply:
🤷‍♂️ I tried, but didn't like how the result reads. `genomeId` is clearer to me, despite not being a real genome id.